ci/Dockerfile.ci-utils

```diff
@@ -1,3 +1,27 @@
+FROM {{ hail_ubuntu_image.image }} AS base
+
+# source: https://cloud.google.com/storage/docs/gsutil_install#linux
+RUN curl --remote-name https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-334.0.0-linux-x86_64.tar.gz && \
+    tar -xf google-cloud-sdk-334.0.0-linux-x86_64.tar.gz && \
+    curl --remote-name https://dl.k8s.io/release/v1.21.14/bin/linux/amd64/kubectl && \
+    install -o root -g root -m 0755 kubectl /usr/local/bin/kubectl
+ENV PATH $PATH:/google-cloud-sdk/bin
+
+# https://bugs.mysql.com/bug.php?id=105288&thanks=sub
```

Comment:
This appears to now be fixed. We should update to latest mysql in a follow up PR. Can you do that?